Packs/ThreatConnect/IncidentFields/incidentfield-Id.json

```diff
@@ -0,0 +1,62 @@
+{
+  "id": "incident_threatconnectid",
```

Comment:
Would be nice to add a description to this field and to the rest of the fields.
Hard to know ID of what this field is

Reply:
I've added description